### ISSUE TITLE
Fighter Bay

### ISSUE THREAD

**Issue by alexanderson1993** [Thorium-Sim/thorium#756]
## Detailed Description

Rotate through maintenance, provide pilots, etc. Base on Magellan Runabouts

**Comment by connorlarsen1** [comment]
Magellan wants to request this function and stuff as well :)

**Comment by alexanderson1993** [comment]
The best way you can help this become a reality is to create descriptions of each of the cards involved (I imagine there will be several) and draw up diagrams of the controls. That'll take care of 30% of the work, and the hardest part of creating new cards. Want to do that for me? 😉

**Comment by connorlarsen1** [comment]
Deal!!!

**Comment by connorlarsen1** [comment]
[Magellan-Thorium.pdf](https://github.com/Thorium-Sim/thorium/files/2776258/Magellan-Thorium.pdf)

**Comment by alexanderson1993** [comment]
I’m afraid I don’t get it. Can you add a few more annotations explaining what all the controls are/do? What is just a graphic and what is an actual control?

**Comment by connorlarsen1** [comment]
Yeah, i'll see what I can do. James designed that one, so i'll get his exact thoughts.

**Comment by alexanderson1993** [comment]
![image1](https://user-images.githubusercontent.com/6558157/58661173-7050ab00-82e4-11e9-9252-fb0cbe52d74d.jpeg)
